packages/carbon-graphs/karma.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (config) {
     },
     singleRun: true,
     files: [
-      './node_modules/@babel/polyfill/dist/polyfill.js',
+      './../../node_modules/@babel/polyfill/dist/polyfill.js',
```

Comment:
Why did we make this change? Is it something we are fixing?

Reply:
actually, when executing karma test cases it is just throwing a warning saying that polyfill is missing in carbon-graphs node modules. When we run npm i command everything is getting installed in global node modules, so I just changed the path and pointed it out to global node modules. 

![image](https://user-images.githubusercontent.com/63670637/102125015-9da17300-3e0e-11eb-9790-588ef14f019c.png)